components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/hybrid/JdbcHybridRoleManager.java

```diff
@@ -313,6 +319,81 @@ public String[] getUserListOfHybridRole(String roleName) throws UserStoreExcepti
         }
     }
 
+    public String[] getPrimaryRoles() throws UserStoreException {
```

Comment:
Shouldn't this be a private method?

Reply:
We are returning a list of usernames from this method. Shall we update the method name to a more meaningful one?

Reply:
> Shouldn't this be a private method?

Will change this to a private method in another PR since this is already merged. Thanks

Reply:
> We are returning a list of usernames from this method. Shall we update the method name to a more meaningful one?

will change it to getPrimaryUsers